static/css/style.css

```diff
@@ -4,7 +4,7 @@ body, input {
     -webkit-font-smoothing: subpixel-antialiased;
     font-family:-apple-system-font, BlinkMacSystemFont, 'Helvetica Neue', sans-serif;
     text-rendering: optimizeLegibility;
-    font-weight:200;
+    font-weight:300;
```

Comment:
this affects mostly numbers - I actually liked them being thin, but I know nothing about design. So, you think it's better this way?

Reply:
Yes, I think it is much better, it was way too thin comparing to the rest of the text.